src/lib/InstantSearch.js

```diff
@@ -8,7 +8,7 @@ import simpleMapping from './stateMappings/simple';
 import historyRouter from './routers/history';
```

Comment:
Inside the file we have a line with:

```js
this._onHistoryChange = function() {};
```

We could also replace it with `noop`.